services/shhext/service.go

```diff
@@ -179,3 +181,46 @@ func (s *Service) Stop() error {
 
 	return s.Service.Stop()
 }
+
+func (s *Service) syncMessages(ctx context.Context, mailServerID []byte, r whisper.SyncMailRequest) (resp whisper.SyncEventResponse, err error) {
+	err = s.w.SyncMessages(mailServerID, r)
```

Comment:
I thought `SyncMessages` returned `(SyncMessagesResponse, error)`, not just `error`, isn't this wrong?

Reply:
It depends which method as there are at least two with the same name :D This one comes from `Whisper` struct and it only returns an error.

Reply:
Makes sense.